examples/dcutr/README.md

```diff
@@ -0,0 +1,42 @@
+## Description
+
+The "Direct Connection Upgrade through Relay" (DCUTR) protocol allows peers in a peer-to-peer network to establish direct connections with each other.
+This example provides a basic usage of this protocol in **libp2p**.
+
+## Usage
+
+To run the example, follow these steps:
+
+1. Run the example using Cargo:
+   ```sh
+   cargo run -- <OPTIONS>
+   ```
+   Replace `<OPTIONS>` with specific options.
+
+### Available Options
+
+The following options can be used with the example:
+
+- `--mode <MODE>`: Specify the mode of operation for the example. Possible values are `dial` (client-dial) or `listen` (client-listen).
+- `--secret-key-seed <SEED>`: Specify a fixed value to generate a deterministic peer ID.
+- `--relay-address <ADDRESS>`: Specify the listening address for the relay server.
+- `--remote-peer-id <ID>`: Specify the Peer ID of the remote peer to establish a connection with (only in `dial` mode).
```

Comment:
This stuff is likely to get stale, I'd suggest we defer the user to use `--help`. What do you think?

Reply:
@thomaseizinger Sure, no problem, just removed that with a small comment to refer to `--help` command.